source/fundamentals/databases-collections.txt

```diff
@@ -0,0 +1,173 @@
+
+Get a List of Collections
+-------------------------
+
+You can query for a list of collections in a database using the
+:java-sync-api:`MongoDatabase.listCollectionNames()
+<com/mongodb/client/MongoDatabase.html#listCollectionNames()>` method:
+
+.. code-block:: java
+
+   for (String name : database.listCollectionNames()) {
+      System.out.println(name);
+   }
+
+Drop a Collection
+-----------------
+
+You can remove a collection from the database using the 
+:java-sync-api:`MongoCollection.drop()
+<com/mongodb/client/MongoCollection.html#drop()>` method:
+
+.. code-block:: java
+
+   MongoCollection<Document> collection = database.getCollection("bass");
+   collection.drop();
+
+.. warning:: Dropping a Collection Deletes All Data in the Collection
+
+   Dropping a collection from your database also implicitly removes all
+   documents that previously belonged to that collection. Only drop
+   collections that contain data that is no longer needed.
+
+Specify Read Preferences, Read Concerns and  Write Concerns
+-----------------------------------------------------------
+
+**Read preferences**, **read concerns**, and **write concerns** control
+how the driver routes read operations and waits for acknowledgement for
+read and write operations when connected to a MongoDB replica set.
+
+Read preferences, read concerns, and write concerns apply to all
+read and write operations, and default settings are shared across entire
+MongoDB instance connections at the MongoClient level.
+
+However, if your application requires a different read preference, read
+concern, or write concern for a ``MongoDatabase`` or ``MongoCollection``,
+you can use the following methods to obtain an instance of a collection
+or database with your preferred preference or concern:
+
+- :java-sync-api:`MongoDatabase.withReadConcern() <com/mongodb/client/MongoDatabase.html#withReadConcern(com.mongodb.ReadConcern)>`
+
+- :java-sync-api:`MongoDatabase.withReadPreference() <com/mongodb/client/MongoDatabase.html#withReadPreference(com.mongodb.ReadPreference)>`
+
+- :java-sync-api:`MongoDatabase.withWriteConcern() <com/mongodb/client/MongoDatabase.html#withWriteConcern(com.mongodb.WriteConcern)>`
+
+- :java-sync-api:`MongoCollection.withReadConcern()<com/mongodb/client/MongoCollection.html#withReadConcern(com.mongodb.ReadConcern)>`
+
+- :java-sync-api:`MongoCollection.withReadPreference() <com/mongodb/client/MongoCollection.html#withReadPreference(com.mongodb.ReadPreference)>`
+
+- :java-sync-api:`MongoCollection.withWriteConcern() <com/mongodb/client/MongoCollection.html#withWriteConcern(com.mongodb.WriteConcern)>`
+
+.. note::
+
+   The ``withReadConcern()``, ``withReadPrefrence()``, and
+   ``withWriteConcern`` methods create a new instance of a
+   ``MongoDatabase`` or ``MongoCollection`` with the desired preference
+   or concern. The ``MongoDatabase`` or ``MongoCollection`` upon which
+   the method is called retains its original preference and concern
```

Comment:
Could use clarification on what the classification of methods is that you're referring to when you say: "the method". Maybe you mean "all instance methods called on ... use the preference and concern"?

Also "original preference" could be more specific.